NMS-14615: Quick Start: Import inventory (#5244)
OpenNMS/opennms: docs/modules/operation/pages/deep-dive/device-config-backup/dcb-requisition.adoc

> It is still possible to retrieve backups on demand by using the Web UI or the REST services.

Reviewer: ```suggestion
It is still possible to retrieve backups on demand by using the web UI or the REST services.
```

Specify a backup schedule in the form of a cron expression to overwrite default behavior.
To prevent all devices fetching the configuration at the same time, it is better to space out the schedule.
For example, `0 0 0 * * ?` runs every 24 hrs at 12:00 a.m., and `0 0 1 * * ?` runs every 24 hrs at 1:00 a.m.

. In the web UI, click the *Gear* symbol at the top-right of the screen.
. Choose menu:Provisioning[Manage Provisioning Requisitions].
. Click *Add Requisition* and enter a name for the requisition.
If you want to enable device config backup on existing nodes in a requisition, select the desired requisition and edit the node definition.
. Click the *Edit* symbol and click *Add Node*.
. Set up your requisition as described in xref:operation:quick-start/inventory.adoc#requisition-create[Create a requisition].
. Add one or more `DeviceConfig` services on the interface where you are backing up the config.
** If a device has multiple config types, use the prefix `DeviceConfig-` to add services.
+
For example, you can use `DeviceConfig-running` to specify a configuration that the device currently has while running.
+

> The UI displays this as "Running Configuration".

Reviewer: ```suggestion
The UI displays this as "Running Configuration."
```

+

> For `DeviceConfig-running-xml`, the UI displays "Running Xml Configuration".

Reviewer: ```suggestion
For `DeviceConfig-running-xml`, the UI displays "Running Xml Configuration."

```

> . Choose *Provisioning>Manage Provisioning Requisitions*.

Reviewer: ```suggestion
. Choose menu:Provisioning[Manage Provisioning Requisitions].
```

. Click *Add Requisition* and enter a name for the requisition.
If you want to enable device config backup on existing nodes in a requisition, select the desired requisition and edit the node definition.

> . Click on the edit icon and click *Add Node*.

Reviewer: ```suggestion
. Click the *Edit* symbol and click *Add Node*.
```

. Set up your requisition as described in xref:operation:quick-start/inventory.adoc#requisition-create[Create a requisition].

> . Add one or more `DeviceConfig` services on the interface where we are backing up the config.

Reviewer: ```suggestion
. Add one or more `DeviceConfig` services on the interface where you are backing up the config.
```

. Click *Add Requisition* and enter a name for the requisition.
If you want to enable device config backup on existing nodes in a requisition, select the desired requisition and edit the node definition.

> . Click on the edit icon and click *Add Node*.

Reviewer: ```suggestion
. Click the *Edit* symbol and click *Add Node*.
```

. Set up your requisition as described in xref:operation:quick-start/inventory.adoc#requisition-create[Create a requisition].
. Add one or more `DeviceConfig` services on the interface where you are backing up the config.

> .. If a device has multiple config types, use the prefix `DeviceConfig-` to add services.

Reviewer: ```suggestion
** If a device has multiple config types, use the prefix `DeviceConfig-` to add services.
```

+

> For example use `DeviceConfig-running` to specify a configuration that the device currently has while running. +

Reviewer: ```suggestion
For example, you can use `DeviceConfig-running` to specify a configuration that the device currently has while running.
+
```
